src/store/reducers/AuthReducer.jsx

```diff
@@ -0,0 +1,15 @@
+export const authReducer = (state, action) => {
+  switch (action.type) {
+    case 'LOGIN': {
+      const { isAuth } = action.payload;
+      localStorage.setItem('isAuth', isAuth);
+      return { ...state, isAuth };
+    }
+    case 'LOAD_FROM_STORAGE': {
+      const isAuth = localStorage.getItem('isAuth') === 'true';
+      return { ...state, isAuth };
+    }
+    default:
+      return state;
+  }
+};
```

Comment:
Another good thing is that you are splitting the things, you have the reducer and the context in their own file, well done